### ISSUE TITLE
Documentatie noemt verkeerd verzend type

### ISSUE THREAD

**Issue by DiderickM** [projectbank/Documentatie#1]
In de documentatie wordt er over GET requets gesporken, dit is echter niet juist. Als ik naar de code kijk wordt er gebruik gemaakt bij add.php van post reqests. Dat is opzich prima, maar het moet nog even in de docs worden aangepast

**Comment by DiderickM** [comment]
Fixed it for the first one

**Comment by Bowero** [comment]
GET and POST requests are intertwined now, because I've mixed them up during the testing procedure. This makes it an issue for the other repository. We will have to look into this before we publish the API. When's that? Monday?

**Comment by DiderickM** [comment]
Yes, we can talk about the API on Monday. I think we can release the documentation but the API is not finished. And mind you that the repo's for this project are already public so technically the API is already released.

**Comment by DiderickM** [comment]
furthermore, since the API is not yet finished and people will only care about this API when they are on project time. I suggest that we release this API on Wednesday when our classmates are on project time.
And if we announce this API on project time then the peer coach can actually help rather than wasting his time.
